docs/source/conf.py

```diff
@@ -21,7 +22,8 @@
 sys.path.insert(0, os.path.abspath("../.."))
 
 project = "movement"
-copyright = "2023, University College London"
+current_year = date.today().year
+copyright = f"2022-{current_year}, University College London"
```

Comment:
As commented above. I would leave this as 2023 everywhere. Otherwise the docs could get out of date with the LICENSE.